examples/example_without_cc_toolchain/MODULE.bazel

```diff
@@ -0,0 +1,22 @@
+"""Bazel module dependencies"""
+
+module(
+    name = "com_google_protobuf-example-without-cc-toolchain",
+    version = "0.0.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "protobuf")
+local_path_override(
+    module_name = "protobuf",
+    path = "../..",
+)
+
+protoc = use_extension("@protobuf//bazel:extensions.bzl", "protoc")
+protoc.prebuilt_toolchain(
+    name = "prebuilt_protoc",
+    # FIXME: users shouldn't be able to vary the version here from the protobuf module
```

Comment:
If we don't want them to vary the version, then why not just statically register the toolchains in `protobuf`s `MODULE.bazel` file? We don't even need a module extension API in that case, just a Starlark flag with which users can toggle between prebuilt and source toolchains via `required_settings`.

Reply:
Thanks, fixed